plugins/mcreator-core/datalists/itemproperties.yaml

```diff
@@ -0,0 +1,26 @@
+- damaged:
+  type: Logic
+  other:

```

Comment:
Logic has true or false states, so there should be no need for min max specified for this type

Reply:
But isnt it seldom true false -> 1/0, if there is some special form of Minecraft where eg. true is 3 and false is 7, generator could handle this on its own, not in datalists